### DISCUSSION TITLE
BreakInfo RankDisplay variable intended usage

### DISCUSSION THREAD

**Discussion by sh0ckR6** [ppy/osu#15185] [Q&A] [answered]
##### Apologies in advance if this is a duplicate! I searched through issues/discussions and came up with nothing

During breaks, I noticed that there was always a mysterious stat called "Rank" that's only ever displayed "-".

What is the intended use for this stat? Is it meant to display your current position on the selected leaderboard? If so, I'm curious as to what issues are blocking this from being implemented, if any?

More specifically, this variable in `osu.Game.Screens.Play.Break`:
https://github.com/ppy/osu/blob/4fcdeffbd9c7bd2c046b943f96f2bfec33c5374e/osu.Game/Screens/Play/Break/BreakInfo.cs#L16
I've noticed that it's only used once for initialization in the same class and never again.

**Comment by peppy** [comment, marked as answer]
I believe that was the intention, or maybe pp or something. We'll probably revisit when there's a design update for that screen.

**Reply by sh0ckR6** [reply]
Makes sense.
Follow-up question: would it be a good idea to just remove the text entirely until it gets revisited? Maybe leave a comment so it's not forgotten when the screen gets revisited for a design update? IMHO it just doesn't feel right to have "information" that doesn't actually do anything.

**Reply by peppy** [reply]
sure, i'm not against it. or just putting something there for now.
